### PR TITLE
Add swagger spec for Microsoft.features

### PR DESCRIPTION
**Pull request by zjpjack**
Add swagger spec for Microsoft.features/operations/read and Microsoft.features/register/action

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by zjpjack** [comment]
@vivsriaus

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json [hunk 1]
```diff
@@ -143,7 +160,7 @@
         "tags": [
           "Features"
         ],
-        "operationId": "Features_Register",
+        "operationId": "SubscriptionFeatures_Register",
```
Comment (vivsriaus) [inline specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: why are we changing this?

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json [hunk 2]
```diff
@@ -176,6 +193,42 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Features/providers/{providerNamespace}/features/{featureName}/register": {
```
Comment (vivsriaus) [inline specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: is this a valid API?

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json [hunk 3]
```diff
@@ -64,6 +64,23 @@
...
+        "tags": [
+          "Features"
+        ],
+        "operationId": "Features_Operations",
+        "description": "Gets all the preview feature operations.",
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array containing metadata about the feature operations. The metadata includes the name of the feature operation, the provider, the resource, the operation, and the description.",
+            "schema": {
+              "$ref": "#/definitions/FeatureOperationResult"
```
Comment (vivsriaus) [inline specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: Shouldn't this be a List response?

File: specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json [hunk 4]
```diff
@@ -2066,6 +2066,23 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Resources/operations": {
+      "get": {
+        "tags": [
+          "Features"
+        ],
+        "operationId": "Features_Operations",
```
Comment (vivsriaus) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json]: fix operationId to reflect this is on Microsoft.Resources

File: specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json [hunk 5]
```diff
@@ -2066,6 +2066,23 @@
...
+        "description": "Gets all the preview feature operations.",
```
Comment (vivsriaus) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json]: Incorrect. this should reflect /operations API of resources

File: specification/resources/resource-manager/Microsoft.Solutions/stable/2017-09-01/managedapplications.json [hunk 6]
```diff
@@ -869,6 +869,23 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/providers/Microsoft.Solutions/operations": {
+      "get": {
+        "tags": [
+          "Features"
+        ],
+        "operationId": "Features_Operations",
```
Comment (vivsriaus) [inline specification/resources/resource-manager/Microsoft.Solutions/stable/2017-09-01/managedapplications.json]: Incorrect. This should be Microsoft.Solutions operations API, not features. Please fix them

**Review by vivsriaus — CHANGES_REQUESTED** [review]
Please fix the operations API for resources, solutions

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-python

**Comment by alvadb** [comment]
Hi, I see that  vivsriaus started the review but there hasn't been any activity for about a week.  Is this PR still valid or can it be closed?

**Comment by alvadb** [comment]
I will close this PR after Friday if there aren't any response or changes.

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json (hunk 2 above)
Comment (zjpjack) [reply specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: We should not have this API.

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json (hunk 1 above)
Comment (zjpjack) [reply specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: Change it back to "Features_Register".

File: specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json (hunk 4 above)
Comment (zjpjack) [reply specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json]: "/providers/Microsoft.Resources/operations" is only supported by apiversion 2015-05-01, but there is no such a folder, do we still need this api in resources.json?

**Pull request closed without merging**